ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -740,13 +740,14 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-registry-s3-cloudfront
-- as: aws-ipi-shared-vpc-phz-f2
-  cron: 12 4 1,3,5,7,9,11,13,15,17,19,21,23,25,27,29 * *
+- as: aws-ipi-shared-vpc-freeipa-phz-f14
```

Comment:
Perhaps use `aws-ipi-shared-vpc-phz-freeipa-f14`, `aws-ipi-shared-vpc-phz` comes from the workflow name.

Reply:
Good catch!

Reply:
@liangxia Thanks for highlighting, Updated the same.